backend/src/test/java/ca/bc/gov/hlth/mohums/integration/MoHUmsIntegrationTests.java

```diff
@@ -262,15 +308,15 @@ public void deleteUserClientRole() {
                 .method(HttpMethod.DELETE)
                 .uri("/users/39f73cbd-dbf0-41c6-a45c-997c44c1c952/role-mappings/clients/db9dd8ab-0f38-4471-b396-e2ddac45a001")
                 .contentType(MediaType.APPLICATION_JSON)
-                .bodyValue("[\n" +
-                        "    {\n" +
-                        "        \"id\": \"a88f491a-3bd1-46ce-9cf6-c509f9a916f8\",\n" +
-                        "        \"name\": \"PSDADMIN\",\n" +
-                        "        \"composite\": false,\n" +
-                        "        \"clientRole\": true,\n" +
-                        "        \"containerId\": \"db9dd8ab-0f38-4471-b396-e2ddac45a001\"\n" +
-                        "    }\n" +
-                        "]")
+                .bodyValue("[\n"
+                        + "    {\n"
+                        + "        \"id\": \"a88f491a-3bd1-46ce-9cf6-c509f9a916f8\",\n"
+                        + "        \"name\": \"PSDADMIN\",\n"
+                        + "        \"composite\": false,\n"
+                        + "        \"clientRole\": true,\n"
+                        + "        \"containerId\": \"db9dd8ab-0f38-4471-b396-e2ddac45a001\"\n"
+                        + "    }\n"
+                        + "]")
```

Comment:
Please avoid including formatting changes in a PR. It makes it harder to figure out what the actual changes are.

(We do need to standardize on our formatting. We have some people in Intellij and some in Netbeans. In the past we've used a Maven-based formatter, but haven't set it up for this project.)

Reply:
Got it. That change slipped through the commit I made :-/